src/main/java/com/pitchain/controller/CompanyController.java

```diff
@@ -0,0 +1,87 @@
+package com.pitchain.controller;
+
+import com.pitchain.common.apiPayload.dto.CustomApiResponse;
+import com.pitchain.dto.req.CreateCompanyReq;
+import com.pitchain.dto.req.LoginCompanyReq;
+import com.pitchain.dto.req.UpdateCompanyReq;
+import com.pitchain.dto.res.CompanyDetailRes;
+import com.pitchain.dto.res.LoginRes;
+import com.pitchain.service.CompanyService;
+import io.swagger.v3.oas.annotations.Operation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+@RequiredArgsConstructor
+@RequestMapping("/companies")
+@RestController
+public class CompanyController {
+
+    private final CompanyService companyService;
+
+    @Operation(summary = "회사 생성")
+    @PostMapping(consumes = {MediaType.MULTIPART_FORM_DATA_VALUE})
+    public CustomApiResponse<Void> createCompany(@RequestPart CreateCompanyReq req) {
+        companyService.createCompany(req);
+        return CustomApiResponse.onSuccess();
+    }
+
+    @Operation(summary = "회사 로그인")
+    @GetMapping("/login")
```

Comment:
HTTP 메서드 변경 놓치신 것 같습니다
```suggestion
    @PostMapping("/login")
```